### ISSUE TITLE
[datadog-plugin-graphql] sampleRates does not work

### ISSUE THREAD

**Issue by xzyfer** [DataDog/dd-trace-js#674]
**Describe the bug**

The `Integration` interface suggests `analytics.sampleRates` can be an object. Given this I would expect the follow set up to set the sample rate for `graphql.parse` and `graphql.validate` events to 0%, and `graphql.execute` to 10%.

```
  tracer.use('graphql', {
    analytics: {
      enabled: false,
      sampleRates: {
        'graphql.parse': 0,
        'graphql.validate': 0,
        'graphql.execute': 0.1,
      },
    },
  })
```

However we're still seeing a high volume `graphql.parse` and `graphql.validate` APM events. 

I've confirmed `extractAnalytics` encounters a `NaN` when trying to set `ANALYTICS_KEY`.

**Environment**

* **Operation system:**
* **Node version:** 11.latest
* **Tracer version:** 0.15.0
* **Agent version:** latest

**Comment by rochdev** [comment]
This was an API we were exploring a little while ago that never made it to be official. It looks like the types were still merged, but the above is not currently supported. We plan to support this, but the API will probably be different.

**Comment by rochdev** [comment]
Fixed the types in 0.15.1 and the functionality will be reintroduced in 0.16.0 with a different API.
